frontend/public/components/monitoring/_monitoring.scss

```diff
@@ -72,6 +72,7 @@
 }
 
 .query-browser__query {
+  height: 33px !important;
```

Comment:
Does shift-enter in the field work properly with this change?

Reply:
Good catch!  Yeah, but not like it did before.  Lemme rethink.

Reply:
Fixed with a better solution.  Thanks, @spadgett.